src/Pages/SignIn/SignIn.scss

```
@@ -0,0 +1,193 @@
+.SignIn {
+  width: 580px;
+  margin: 0 auto;
+  background-color: white;
+  font-family: notokr-medium;
+
+  header {
+    display: flex;
+    justify-content: center;
+  }
+
+  main {
+    p {
+      text-align: center;
+      font-size: 20px;
+      em {
+        color: #009ee3;
+        font-style: normal;
+      }
+    }
+
+    div {
+      .memberSignIn {
+        border: 1px solid #d9d9d9;
+
+        .tab {
+          padding-left: 0px;
+          margin-top: 0px;
+          display: inline;
+          list-style: none;
+          display: flex;
+          li {
+            width: 50%;
+            height: 68px;
+            border-left: none;
+            border-right: none;
+            .memberTab {
+              position: relative;
+              top: 22px;
+              left: 94px;
+            }
+            .NonmemberTab {
+              position: relative;
+              top: 22px;
+              left: 40px;
+            }
+          }
+          .selectedTabs {
+            border: 1px solid #d9d9d9;
+            border-top: 3px solid #009ee3;
+            border-bottom: none;
+            color: #009ee3;
+            font-size: 20px;
+          }
+          .unselectedTabs {
+            border: 1px solid #d9d9d9;
+            border-top: none;
+            font-size: 20px;
+            color: #8f8f8f;
+            background: #f0f0f0;
+          }
+        }
+        .idpw {
+          padding: 40px 30px;
+          .input {
+            display: flex;
+            flex-direction: column;
+            .account {
+              height: 50px;
+              margin-top: 6px;
```

```suggestion
        .idpw {
          padding: 40px 30px;
          
          .input {
            display: flex;
            flex-direction: column;
            
            .account {
              height: 50px;
              margin-top: 6px;
```
구분되는 요소들끼리는 한줄씩 띄워주세요